finagle-core/src/main/scala/com/twitter/finagle/param/Params.scala

```diff
@@ -123,16 +123,31 @@ case class HighResTimer(timer: com.twitter.util.Timer) {
 
 object HighResTimer {
 
+  /**
+   * The underlying default Timer used for configuration
+   * Default `stop` behaviour is ignored in order to not be accidentally stopped
+   */
+  private[this] val underlying = new JavaTimer(true, Some("HighResTimer")) {
+    override def stop(): Unit = ()
+    def stopTimer(): Unit = super.stop()
+  }
+
   /**
    * The default Timer used for configuration.
    *
    * It is a shared resource and as such, `stop` is ignored.
    */
-  val Default: com.twitter.util.Timer =
-    new JavaTimer(true, Some("HighResTimer")) {
-      override def stop(): Unit = ()
-    }
+  val Default:com.twitter.util.Timer = underlying
 
+  /**
+   * Stop default HighResTimer
+   */
+  def stop():Unit = {
+    util.DefaultLogger.warning("Stopping the default Finagle HighResTimer. When timer is stopped, " +
```

Comment:
instead of util.DefaultLogger, let's use a similar Logger.get() approach to what we did in the other review.